src/test/resources/ohio.xml

```diff
@@ -0,0 +1,86 @@
+<record>
```

Comment:
I think in general, the inline xml way of writing tests will be more useful, because many mappings have branches in them depending on what the document has, and you can't test that situation with a single example file. 

If that situation arises with Ohio, you're going to end up with a hybrid of test data. Just thinking it might make sense to start with the inline xml approach.